linux_os/guide/system/logging/log_rotation/ensure_logrotate_activated/oval/shared.xml

```diff
@@ -1,16 +1,19 @@
 <def-group>
-  <definition class="compliance" id="ensure_logrotate_activated" version="1">
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
     {{{ oval_metadata("
       The frequency of automatic log files rotation performed by the logrotate utility should be configured to run daily
       ") }}}
     <criteria comment="/etc/logrotate.conf contains daily setting and /etc/cron.daily/logrotate file exists" operator="AND">
+      <extend_definition comment="package logrotate installed" definition_ref="package_logrotate_installed" />
```

Comment:
I wonder if this `extended_definition` is necessary when the rule has `platform: package[logrotate]`